docs/designs/conceptual-versioning.md

```diff
@@ -366,48 +367,30 @@ _site/en-us/group-01ddf122/dotnet/api/system.string/index.html
 
 - Content for `.manifest.json`
 
-    ##### Ideal output
-
-    ```json
-    {
-        "files":[
-            {
-                "siteUrl": "/{SitePath}",
-                "outputPath": "{outputPath}",
-                "sourcePath": "{sourcePath}",
-                "monikers": [
-                    "moniker1",
-                    ...
-                ]
-            },
-        ]
-    }
-    ```
-
-    ##### Legacy output
-
-    ```json
-    {
-        "groups":[
-            "group": "{groupId}",
+```json
+{
+    "groups":{
+        "{groupId}": {
             "monikers": [
                 "moniker1",
                 ...
             ],
             ...
-        ],
-        "files":[
-            {
-                "siteUrl": "{SitePath}",
-                "outputPath": "{outputPath}",
-                "sourcePath": "{sourcePath}",
-                "group": "{groupId}"
-            },
-        ]
-    }
-    ```
+        },
+        ...
+    },
+    "files":[
+        {
+            "siteUrl": "{SitePath}",
+            "outputPath": "{outputPath}",
+            "sourcePath": "{sourcePath}",
+            "group": "{groupId}"
```

Comment:
if the group is part of siteUrl and outputPath, do we still need it here?

Reply:
yes, it is a part of `outputPath`, but it will be a little bit hard to extract it from `outputPath`, so we keep it here to avoid unnecessary extract

Reply:
that means we need it during publishing, got it